setup.py

```diff
@@ -234,9 +234,10 @@ def get_version_tofu(path=_HERE):
 
 # Get the long description from the README file
 # Get the readme file whatever its extension (md vs rst)
+
 _README = [
     ff
-    for ff in os.listdir(os.path.abspath(os.path.dirname(__file__)))
```

Comment:
In principle, this should not change the result, since _HERE = os.path.abspath(os.path.dirname(__file__))
as defined above (line 220)

Does it really change something ? If so, does someone understand why ?

Reply:
it doesn't but it's better

Reply:
👍